src/controller/RaceController.js

```diff
@@ -0,0 +1,55 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import RaceView from '../view/RaceView.js';
+import CarModel from '../model/CarModel.js';
+import RaceModel from '../model/RaceModel.js';
+import InputValidator from './InputValidator.js';
+import { FORWARD_THRESHOLD } from '../constants/constraints.js';
+
+class RaceController {
+  constructor() {
+    this.view = new RaceView();
+    this.race = null;
+  }
+
+  async startRace() {
+    const carNames = await this.view.getCarNames();
+    InputValidator.validateCarName(carNames);
+    const attemptCount = await this.view.getAttemptCount();
+    InputValidator.validateAttemptCount(attemptCount);
+
+    const cars = carNames.map((name) => new CarModel(name));
+    this.race = new RaceModel(cars);
+
+    this.runRace(attemptCount);
+    this.showWinners();
+  }
+
+  runRace(attemptCount) {
+    Console.print('\n실행 결과');
```

Comment:
Controller에서 `Console`을 직접 호출하기 보다는 view 레이어를 한 번 거치면 좋을 거 같아요!

Console과 Controller에 직접적인 의존성을 없애고 view만 의존하게 만들 수 있을 거 같아요

```suggestion
    this.view.printRacingResultInfo();
```

Reply:
혼자 저렇게 있어서 찜찜했는데, 그런 방법이 있군요 !!
철처히 분리를 해야했었는데 감사합니다~!!